src/scss/_variables.scss

```diff
@@ -1,3 +1,10 @@
+////
+/// @group o-business-card
+/// @link http://registry.origami.ft.com/components/o-business-card
+////
+
 /// Silent mode
 /// @type Bool
 $o-business-card-is-silent: true !default;
+
+@include oColorsSetUseCase(o-business-card, border, pink-tint2);
```

Comment:
There is a fallback in case the background isn't set, so not sure why we'd need to set it?